src/components/shared/accountMigration/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { withTranslation } from 'react-i18next';
+import CopyToClipboard from '@toolbox/copyToClipboard';
+import Icon from '@toolbox/icon';
+import LiskAmount from '@shared/liskAmount';
+import AccountVisualWithAddress from '@shared/accountVisualWithAddress';
+import styles from './index.css';
+
+const AccountMigration = ({ t, account, showBalance }) => (
+  <div className={styles.accountContainer}>
+    <div>
+      <h5>{t('Old account')}</h5>
+      <div className={styles.addressContainer}>
+        <AccountVisualWithAddress address={account.legacy?.address} truncate={false} />
+        <CopyToClipboard type="icon" value={account.legacy?.address} copyClassName={styles.copyIcon} />
+      </div>
+      {showBalance && (
+        <p>
+          <span>{`${t('Balance')}: `}</span>
+          <LiskAmount val={parseInt(account.legacy?.balance, 10)} token="LSK" />
```

Comment:
We don't need to use `parseInt`. You can convert to a number using the `Number` constructor

Reply:
Fixed